frontend/src/citizen-frontend/calendar/DayView.tsx

```diff
@@ -70,122 +79,128 @@ export default React.memo(function DayView({
+                text={i18n.common.save}
+                iconRight
+              />
+            ) : (
+              <InlineButton
+                icon={faPen}
+                onClick={startEditing}
+                text={i18n.common.edit}
+                iconRight
+              />
+            )
+          ) : null}
+        </ReservationTitle>
+        <Gap size="s" />
+        {editing
+          ? editorState.map(({ child, startTime, endTime, errors }, index) => (
+              <Fragment key={child.id}>
+                {index !== 0 ? <Separator /> : null}
+                <H3 noMargin>
+                  {child.preferredName || child.firstName.split(' ')[0]}
+                </H3>
+                <Gap size="s" />
+                <Grid>
+                  <Label>Varaus</Label>
+                  <InputWrapper>
+                    <InputField
+                      type="time"
+                      onChange={editorStateSetter(index, 'startTime')}
+                      value={startTime}
+                      info={errorToInputInfo(
+                        errors.startTime,
+                        i18n.validationErrors
+                      )}
+                      readonly={saving}
+                    />
+                    –
+                    <InputField
+                      type="time"
+                      onChange={editorStateSetter(index, 'endTime')}
+                      value={endTime}
+                      info={errorToInputInfo(
+                        errors.endTime,
+                        i18n.validationErrors
+                      )}
+                      readonly={saving}
+                    />
+                  </InputWrapper>
+                  <Label>Toteuma</Label>
+                  <span>–</span>
+                </Grid>
+              </Fragment>
+            ))
+          : childrenWithReservations.map(({ child, reservation }, index) => (
+              <Fragment key={child.id}>
+                {index !== 0 ? <Separator /> : null}
+                <H3 noMargin>
+                  {child.preferredName || child.firstName.split(' ')[0]}
+                </H3>
+                <Gap size="s" />
+                <Grid>
+                  <Label>Varaus</Label>
+                  {reservation?.absence ? (
+                    <span>
+                      {i18n.calendar.absences[reservation.absence] ??
+                        i18n.calendar.absent}
+                    </span>
+                  ) : reservation?.reservation ? (
+                    <span>{`${reservation.reservation.startTime} – ${reservation.reservation.endTime}`}</span>
+                  ) : (
+                    <NoReservation>Ei varausta</NoReservation>
+                  )}
+                  <Label>Toteuma</Label>
+                  <span>–</span>
+                </Grid>
+              </Fragment>
+            ))}
+        {confirmationModal ? (
+          <InfoModal
+            title="Haluatko tallentaa muutokset?"
```

Comment:
Should we already have some i18n for these?

Reply:
Probably, yeah.